build/common/installer/scripts/tomlparser-prom-agent-config.rb

```diff
@@ -63,6 +72,28 @@ def populateSettingValuesFromConfigMap(parsedConfig)
           puts "Using config map value: AZMON_SIDECAR_FBIT_MEM_BUF_LIMIT = #{@promFbitMemBufLimit.to_s + "m"}"
         end
       end
+
+      network_listener_waittime_config = parsedConfig[:agent_settings][:network_listener_waittime]
```

Comment:
this change will not work as this configmap only invoked when prometheus side car. this needs to be some separate config file.

Reply:
Prometheus side car is the only place where we missed. I think this works.

Reply:
its my bad. I thought you removed prev code. I see this code there in both configs and should be good.